lib/menu.ts

```diff
@@ -301,12 +318,27 @@ function findAncestorChild(ancestor: Element, elem: Element|null): Element|null
  * Sets the width of the given menu element to match the given container element. Used by
  * IMenuOptions setting 'stretchToSelector'.
  */
-function stretchMenuToContainer(menuElem: HTMLElement, containerElem: Element): void {
-  const style = menuElem.style;
+function stretchMenuToContainer(menuEl: HTMLElement, containerElem: Element): void {
```

Comment:
Why the rename? :)

Reply:
Tslint complained about shadowed variable name (there is a top-level function by the same name).